scheduler/roles/slurm/handlers/main.yml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- name: Restart munge
+  ansible.builtin.systemd_service:
+    name: munge
+    state: restarted
+
+- name: Restart mysqld
+  ansible.builtin.systemd_service:
+    name: mysqld
+    state: restarted
+  run_once: true
+  when: restart_slurm_services
+  delegate_to: "{{ slurm_db_host }}"
+
+- name: Restart mariadb
+  ansible.builtin.systemd_service:
+    name: mariadb
+    state: restarted
+  run_once: true
+  when: restart_slurm_services
+  delegate_to: "{{ slurm_db_host }}"
+
+- name: Reload slurmdbd
+  ansible.builtin.systemd_service:
+    name: slurmdbd
+    state: reloaded
+    daemon_reload: true
+    enabled: true
+  when: restart_slurm_services and ('slurm_dbd' in group_names)
```

Comment:
cant we use block and when condition restart_slurm_services . also inside block with groups?